### ISSUE TITLE
Some AFR articles not working only load first 2 paragraphs 

### ISSUE THREAD

**Issue by blueys45** [bpc-clone/bypass-paywalls-chrome-clean#105]
Describe issue/request: Some AFR articles are not working with the BPC V 3.7.1.0 using the kiwi browser. They only load the top two paragraphs.

Link of paywalled article: https://www.afr.com/world/asia/china-s-broken-housing-market-and-a-generation-lying-flat-20240612-p5jl49

Actions you tried yourself and if it works (clear cookies, disabe Javascript, change user-agent/referer, reader view, archive sites and/or add a custom site): Tried to reinstall the extension, clear cache and cookies but still encounter the same issue.

Browser (main version): Kiwi Browser

BPC-extension version (number): 3.7.1.0

PS always first see changelog and search (domain) name in (all) issues (& use latest ext. version).
Also request host permission for unlisted domain (by clearing cookies once by BPC-icon; X = no fix).
![Screenshot_20240619-092648](https://github.com/bpc-clone/bypass-paywalls-chrome-clean/assets/56347834/e6395606-59ef-40ac-a10c-11379f506299)

**Comment by LukePrior** [comment]
same here: https://www.afr.com/companies/energy/over-budget-and-plagued-with-delays-uk-nuclear-lessons-for-australia-20240621-p5jnkq

**Comment by RH383** [comment]
Same here….was working yesterday, but only some articles are loading in full. Im using the latest version of BPC, btw. Thanks

**Comment by RH383** [comment]
Just for my own information how can I enable automatic updates?

**Comment by speedboattoyota** [comment]
Hi - I noticed this issue occurring again for afr articles. I am on version 3.8.3.1 on chrome.

Example link:
https://www.afr.com/technology/how-these-parents-keep-their-kids-off-social-media-20240910-p5k9aa

**Comment by RH383** [comment]
Yes can concur the issue is still persisiting. Thanks for the heads up. I am using Edge and Chrome, and using the latest BPC version,
